app/scripts/lib/ppom/ppom-middleware.test.ts

```diff
@@ -0,0 +1,91 @@
+import { createPPOMMiddleware } from './ppom-middleware';
+
+Object.defineProperty(globalThis, 'fetch', {
+  writable: true,
+  value: () => undefined,
+});
+
+Object.defineProperty(globalThis, 'performance', {
+  writable: true,
+  value: () => undefined,
+});
+
+describe('PPOMMiddleware', () => {
```

Comment:
Should we also add a test with a method that is not whitelisted for ppom (method not in ConfirmationMethods) and ensure that usePPOM is not executed?